build.gradle

```diff
@@ -1,8 +1,8 @@
 plugins {
-	id 'fabric-loom' version '0.11.+'
+	id 'fabric-loom' version '0.12-SNAPSHOT'
```

Comment:
Ok, I might be a tad annoying with this, but could we switch this to quilt loom instead?
Given it should still produce valid Fabric JARs it would be better than the little mess of plugins right here.

Reply:
I'm not very experienced with quilt yet, so maybe you could do that.